imports/ui/Body.js

```diff
@@ -60,12 +59,23 @@ Template.Ranking.helpers({
 				break;
 			default:
 				console.error("Format '" + format + "' is invalid");
-				break;
 		}
 
 		return league + " " + Constants.MAHJONG_CLUB_LEAGUE;
 	},
-	getPlayers(sortBy) {
-		return Players.find({}, { "sort": sortBy });
+	getPlayers(sortBy, format) {
+		let hasPlayedGame = {};
+		switch (format) {
+			case Constants.GAME_TYPE.JAPANESE:
```

Comment:
Also this is not my style for switch/case.  Please remove the extraneous indentation.